### PR TITLE
[PHPStan] level 2 fixes

### PR DESCRIPTION
**Pull request by weisswurstkanone**

### CODE REVIEW

File: bundles/AdminBundle/Controller/Rest/Element/DataObjectController.php [hunk 1]
```diff
@@ -20,6 +20,7 @@
 use Pimcore\Model\DataObject;
 use Pimcore\Model\Webservice\Data\DataObject\Concrete\In as WebserviceObjectIn;
 use Pimcore\Model\Webservice\Data\DataObject\Concrete\Out as WebserviceObjectOut;
+use Pimcore\Model\Webservice\Data\DataObject\Concrete\Out;
```
Comment (jdreesen) [inline bundles/AdminBundle/Controller/Rest/Element/DataObjectController.php]: This is `use`d twice. See also line 133.

Comment (weisswurstkanone) [reply bundles/AdminBundle/Controller/Rest/Element/DataObjectController.php]: thx! 👍

File: bundles/AdminBundle/Controller/Rest/Element/DataObjectController.php [hunk 2]
```diff
@@ -125,10 +125,11 @@ public function getAction(Request $request, $id = null)
             $stopwatch->start('ws', $profileName);
         }
 
-        /** @var WebserviceObjectOut|WebserviceFolderOut $out */
         if ($object instanceof DataObject\Folder) {
+            /** @var WebserviceObjectOut|WebserviceFolderOut $out */
             $out = $this->service->getObjectFolderById($id);
         } else {
+            /** @var WebserviceObjectOut|Out $out */
```
Comment (jdreesen) [inline bundles/AdminBundle/Controller/Rest/Element/DataObjectController.php]: Not that the `use` statement is removed, this should be removed, too.

```suggestion
            /** @var WebserviceObjectOut $out */
```

File: models/Document/Tag/Renderlet.php [hunk 3]
```diff
@@ -177,6 +177,7 @@ public function frontend()
                 Logger::error($e);
             }
         }
+        return null;
```
Comment (brusch) [inline models/Document/Tag/Renderlet.php]: There's actually no return value expected, so `void` would be best I'd say.

File: models/Document/Tag/Snippet.php [hunk 4]
```diff
@@ -177,6 +177,7 @@ public function frontend()
                 return 'ERROR: ' . $e->getMessage() . ' (for details see log files in /var/logs)';
             }
         }
+        return null;
```
Comment (brusch) [inline models/Document/Tag/Snippet.php]: Here `void` as well.

File: models/Document/Tag/Wysiwyg.php [hunk 5]
```diff
@@ -203,5 +203,6 @@ public function rewriteIds($idMapping)
 
         $html->clear();
         unset($html);
+        return null;
```
Comment (brusch) [inline models/Document/Tag/Wysiwyg.php]: Also no return value expected here, so `void`;

**Pull request merged**